home/views/api/admin.py

```diff
@@ -576,3 +576,37 @@ def get(self, request, *args, **kwargs):
             return response
         else:
             return HttpResponse(status=401)
+
+
+class AdminUsersByAgeGroupView(View):
+    http_method_names = ["get"]
+
+    def get(self, request, *args, **kwargs):
+        if request.user.is_authenticated:
+            contest_id = request.GET.get("contest_id", None)
+            if contest_id is None:
+                return HttpResponse(status=422)
+            contest = Contest.objects.get(pk=contest_id)
+            age_min = request.GET.get("age_min", None)
+            age_max = request.GET.get("age_max", None)
+            with connection.cursor() as cursor:
+                cursor.execute(
+                    """
+                    SELECT COUNT(*)
+                    FROM (
+                        SELECT home_account.id AS id, home_account.age AS age FROM home_account
+                        JOIN home_dailywalk ON home_account.id=home_dailywalk.account_id
```

Comment:
I don't think joining on the Daily Walks table is necessary or appropriate in this query, and is probably throwing the counts off.

If we have start/end dates, then we should query for accounts created between those start/end dates.

If we have a contest_id, then we should query for accounts that are associated with that contest (which can include accounts that signed up before the contest started).